deploy/backup/restore.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: crunchydata.com/v1
+kind: Pgtask
+metadata:
+  labels:
+    pg-cluster: cluster1
+    pgouser: admin
+  name: cluster1-backrest-restore
+  namespace: pgo
+spec:
+  name: cluster1-backrest-restore
+  namespace: pgo
+  parameters:
+    backrest-restore-from-cluster: cluster1
+    backrest-restore-opts: --type=time --target="2021-04-16 15:13:32"
```

Comment:
show we keep that time?

Reply:
it is just a template.  We can add something like `--target="<target_time>"` but in this case we do not show the format.